book/invalidation.md

```diff
@@ -2244,7 +2243,7 @@ def paint_visual_effects(node, cmds, rect):
 However, the `font` method needs a little bit of work. Until now,
 we've read the node's `style` and passed that to `font`:
 
-``` {.python expected=False}
+``` {.python .example}
```

Comment:
This is supposed to show existing (a.k.a. old) code, to be replaced in later code blocks. Perhaps this should have the gray background, though I'm willing to go either way.

Reply:
It shows real browser code so let's not add example